src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1686,11 +1704,19 @@ public int getStartIndex() {
     }
 
     public String getCourseId() {
-        return sanitizeForHtml(bundle.feedbackSession.courseId);
+        if (displayableCourseId == null) {
+            displayableCourseId = sanitizeForHtml(bundle.feedbackSession.courseId);
+        }
+        
+        return displayableCourseId;
     }
 
     public String getFeedbackSessionName() {
-        return sanitizeForHtml(bundle.feedbackSession.feedbackSessionName);
+        if (displayableFsName == null) {
+            displayableFsName = sanitizeForHtml(bundle.feedbackSession.feedbackSessionName);
+        }
```

Comment:
Should these be done in `initCommonVariables()`?
I'm not sure how many times these are called on the front-end, but it doesn't seem effective having to do an if check each time.